### PR TITLE
verified epoch logging

### PR DESCRIPTION
**Pull request by williamFalcon**
ref: fix epoch logging

### CODE REVIEW

**Comment by williamFalcon** [comment]
@ananthsub ... i can't find any issues

File: tests/base/boring_model.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    def training_step(self, batch, batch_idx):
+        output = self.layer(batch)
+        loss = self.loss(batch, output)
+        return {"loss": loss}
+
+    def training_step_end(self, training_step_outputs):
+        return training_step_outputs
+
+    def training_epoch_end(self, outputs) -> None:
+        torch.stack([x["loss"] for x in outputs]).mean()
```
Comment (ananthsub) [inline tests/base/boring_model.py:62]: did you mean to log the avg loss here?

File: tests/base/boring_model.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    def validation_step(self, batch, batch_idx):
+        output = self.layer(batch)
+        loss = self.loss(batch, output)
+        return {"x": loss}
+
+    def validation_epoch_end(self, outputs) -> None:
+        torch.stack([x['x'] for x in outputs]).mean()
```
Comment (ananthsub) [inline tests/base/boring_model.py:70]: ditto

**Review by ananthsub — APPROVED** [review]
